src/Compilers/VisualBasic/VbcCore/vbc.cmd

```diff
@@ -0,0 +1,2 @@
+echo off
+%~dp0\corerun.exe %~dp0\vbc.exe %*
```

Comment:
We will, but right now on Linux we still use a "setup" script in cibuild.sh. I'm currently testing whether or not we can get rid of that, but it will be dependent on verifying that all the Linux NuGet packages are now authored correctly and don't need any patching.